core/domain/caching_services.py

```diff
@@ -134,12 +139,14 @@ def _get_memcache_key(namespace, sub_namespace, obj_id):
         sub_namespace_key_string, MEMCACHE_KEY_DELIMITER, obj_id)
 
 
-def flush_memory_caches():
+def flush_memory_caches() -> None:
     """Flushes the memory caches by wiping all of the data."""
-    memory_cache_services.flush_caches()
+    memory_cache_services.flush_caches() # type: ignore[attr-defined]
```

Comment:
Why is this ignore added?

Reply:
To fix this you might need to add `memory_cache_services` to the mypy_imports.py and the import it here just in the MYPY mode.

Reply:
done